Protocols/public_repos.md

```diff
@@ -4,5 +4,44 @@ title: Public Repos
 parent: Protocols
 nav_order: 1
 ---
+**Last Updated:** 2021-05-11
+# Protocol for Public Repositories
 
-TODO: Fill in requirements and steps to creating public repository
+There are 3 kinds of repositories we host:
+
+ **_private_ :** Only visible to other lab members. Useful for sharing quick research code or incomplete ideas.
+
+ **_public experimental_ :** Minimal guarantees. Likely corresponds to a publication and might not be maintained moving forward.
+
+ **_public stable_ :** Guaranteed to adhere to highest repo standards. Tests with good coverage. Merges accepted through PRs that pass CI. Good target to fork.
+
+
+## Repo Requirements
+
+### 🌱 **_private_**
+- no requirements.
+
+### 🌿 **_public experimental_** 🧪
+- Tidy code on `main/master`. No extraenous files that play no role in the functionality
```

Comment:
Typo: **extraneous**

Reply:
great! thank you for the review. I assume this means you are cool with the proposed criteria as well?